protocol/avdl/keybase1/chat_local.avdl

```diff
@@ -73,6 +73,7 @@ protocol chatLocal {
   }
 
   record ConversationInfoLocal {
+    chat1.ConversationID id;
```

Comment:
We should probably block this on that since otherwise there will be a
protocol incompatibility.

On Wednesday, August 31, 2016, Song Gao <notifications@github.com> wrote:

> In protocol/avdl/keybase1/chat_local.avdl
> <https://github.com/keybase/client/pull/4120#discussion_r77015921>:
>
> > @@ -73,6 +73,7 @@ protocol chatLocal {
> >    }
> >
> >    record ConversationInfoLocal {
> > +    chat1.ConversationID id;
>
> Sure! Would that be a separate ticket?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/keybase/client/pull/4120/files/5097d46a43f458aff69c38f96684c2076242af1e#r77015921>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA05_1NV2jBMzGYKE87KQHBUfEdfHa3bks5qlaSggaJpZM4JxGwx>
> .
>